docs/user-guide/area_calc.ipynb

```diff
@@ -6,9 +6,6 @@
    "source": [
```

Comment:
<p>Can omit: "For more details on how to load in data, check out our&nbsp;<a href="https://uxarray.readthedocs.io/en/latest/examples/001-read-grid-data.html" target="_blank">previous usage example</a>"</p><p> </p><p>Can we also remove section 6, since we don't really advertise any of our functionality to work without our data structures? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UXARRAY/uxarray/pull/788/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>